proto/prehog/v1alpha/teleport.proto

```diff
@@ -818,6 +843,26 @@ enum FeatureRecommendationStatus {
   FEATURE_RECOMMENDATION_STATUS_DONE = 2;
 }
 
+// LicenseLimitEvent is emitted when access to Teleport feature
+// is denied based on license limits
+//
+// PostHost event: tp.license.limit
+message LicenseLimitEvent {
+  // PostHost property: tp.license_limit
+  LicenseLimit license_limit = 1;
+}
+
+// License limit
+enum LicenseLimit {
+  LICENSE_LIMIT_UNSPECIFIED = 0;
+  // LICENSE_LIMIT_DEVICE_TRUST_TEAM_JAMF is emitted if license does not
+  // allow Jamf integration (e.g. Team Plan)
+  LICENSE_LIMIT_DEVICE_TRUST_TEAM_JAMF = 1;
```

Comment:
```suggestion
  DEVICE_LICENSE_LIMITS_JAMF_NOT_ALLOWED = 1;
```

Reply:
Keeping this and below as is. see https://github.com/gravitational/teleport/pull/31450#discussion_r1323478062